lib/pages/fragments/SearchBar.dart

```diff
@@ -102,6 +103,18 @@ class _SearchBarState extends State<SearchBar> {
     });
   }
 
+  String _normalizeGreek(String text){
+    text = text.replaceAll(new RegExp("Ά|Α|ά"), 'α')
+        .replaceAll(new RegExp("Έ|Ε|έ"), 'ε')
+        .replaceAll(new RegExp("Ή|Η|ή"), 'η')
+        .replaceAll(new RegExp("Ί|Ϊ|Ι|ί|ΐ|ϊ"), 'ι')
+        .replaceAll(new RegExp("Ό|Ο|ό/"), 'ο')
```

Comment:
Is this supposed to be here?

```suggestion
        .replaceAll(new RegExp("Ό|Ο|ό"), 'ο')
```

Reply:
oh wow! Good catch!